Tests/RegexDSLTests/RegexDSLTests.swift

```diff
@@ -171,4 +172,28 @@ class RegexDSLTests: XCTestCase {
     try run(regex)
     try run(regexLiteral)
   }
+  
+  func testAllMatches() throws {
+    let line = """
+      A6F0..A6F1    ; Extend # Mn   [2] BAMUM COMBINING MARK KOQNDON..BAMUM COMBINING MARK TUKWENTIS
+      """
+
+    let regexLiteral = try MockRegexLiteral(
+      #"([0-9A-F]+)(?:\.\.([0-9A-F]+))?\s+;\s+(\w+)"#,
+      capturing: (Substring, Substring?, Substring).self)
+    
+    let lineAll = line.allMatches(regexLiteral)
+    XCTAssertEqual(1, lineAll.count)
```

Comment:
Nit: Most of this repo is doing `XCTAssertEqual(actual, expected)` instead.

Reply:
Ah, thanks! Didn't realize — my default is expected/actual.